### PR TITLE
Add join call button to conversation navigation bar if there is an ongoing call

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

Add join call button to conversation navigation bar if there is an ongoing call:

simulator screen shot 05 07 2017 10 36 43

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift [hunk 1]
```diff
@@ -174,6 +175,22 @@ public extension ConversationViewController {
         
         return items
     }
+
+    private func shouldShowCollectionsButton() -> Bool {
+        guard #available(iOS 8.3, *) else { return false }
```
Comment (vytis) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift]: Why it is disabled for iOS 8.0-8.2?

Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift]: I only moved this part to have all button related code next to each other, but as far as I can recall there were some issues with the collection view. However I will remove this check as we dropped iOS 8 altogether now.

Comment (vytis) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift]: Oh yeah, I always forget that it really happened 😢

Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift]: @vytis updated!

**Pull request merged**